src/main/java/codeu/controller/BotHandlerInterface.java

```diff
@@ -0,0 +1,17 @@
+package codeu.controller;
+
+import codeu.model.data.BotRequest;
+
+/**
+ * The interface which is implemented by each bot type in order to produce the desired
+ * output. Every bot have a bot-specific handler which implements BotHandlerInterface.
+ */
+public interface BotHandlerInterface {
+  /**
+   * Handles BotRequests. When a bot is triggered and a BotRequest created, the handler
```

Comment:
Why does the handler return a String if it does the conversation location and sending?

Should instead this be a `public void hander(BotRequest request)`?

IMHO, I think it's fair for the handler to decide what it means to "handle" the request, since in some cases that might be > 1 response, or it might be asynchronously handled so that say you have a command `/launch rocket` the request wouldn't wait to return to the chat room while the rocket took minute to hours to launch. It would start the launch sequence and post updates as it progressed